binarySearch/binarySearch.js

```diff
@@ -11,5 +11,30 @@
 **/
 
 const binarySearch = (nums, target) => {
+    let min = 0;
+	let max = nums.length - 1;
```

Comment:
Will do that. Still not really solid on callbacks. Will read, practice more and re-submit the 'CallBack Practice' challenge for review. Thanks for the feedback